content/docs/technical_practices/azure/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+# Page title as it appears in the navigation menu
+title: "Azure"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 6
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Azure
```

Comment:
see https://github.com/alan-turing-institute/REG-handbook/pull/107#discussion_r1234163867

Reply:
Mathison is referenced later in this same page